modules/installation-requirements-user-infra.adoc

```diff
@@ -181,6 +181,7 @@ On your z/VM instances, set up:
 * 3 guest virtual machines for {product-title} control plane machines, one per z/VM instance
 * At least 6 guest virtual machines for {product-title} compute machines, distributed across the z/VM instances
 * 1 guest virtual machine for the temporary {product-title} bootstrap machine
+* To ensure the availability of integral components in an overcommitted environment, increase the priority of the control plane using the cp command SET SHARE. Do the same for infrastructure plane machines if they exist. See link:https://www.ibm.com/support/knowledgecenter/en/SSB27U_7.1.0/com.ibm.zvm.v710.hcpb7/setshare.htm[SET SHARE] in the IBM Knowledge Center.
```

Comment:
If SET SHARE is a command, it should be enclosed in backticks (``):
> using the cp command `SET SHARE`.

Also, on the page you've linked, they capitalize as "CP commands" so maybe we should here as well?